packages/forma-36-react-components/tools/webpack.production.config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
       },
     },
   },
+  entry: {
+    main: './src/index.ts',
+    datePicker: './src/components/DatePicker/index.ts',
```

Comment:
@m10l @suevalov ok so we could add 2 entry points for datePicker and timeZone select or i'll bundle those in date.js (date-utils.js). wdyt?